.gitignore

```diff
@@ -42,5 +42,8 @@ cacheKey.expiration
 #Goland IDE
 .idea/
 
+# VSCode editor
```

Comment:
Would you remind moving this section? You'll find we actually already added this one in on line 17 as part of the most recent merge. GMTA :)

Reply:
Thanks James.  I've made the requested changes.